Modules/Core/SpatialObjects/test/itkSpatialObjectToImageFilterTest.cxx

```diff
@@ -70,6 +70,12 @@ itkSpatialObjectToImageFilterTest(int, char *[])
   imageFilter->SetChildrenDepth(childrenDepth);
   ITK_TEST_SET_GET_VALUE(childrenDepth, imageFilter->GetChildrenDepth());
 
+  ImageType::IndexType indx;
+  indx[0] = 10;
+  indx[1] = 10;
+  imageFilter->SetIndex(indx);
```

Comment:
Doesn't this necessitate changes further down in the test?

Reply:
That is what I initially thought too, but changing the index of a region only changes the spatial extent covered by it, and the physical and index position of the "object" spanned by that region is unchanged.  So, the test was actually already verifying the usage of a region's index in the most thorough way (writing data in physical space and verifying it in index space), and yet no changes were needed.